src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -621,10 +605,16 @@ private FingerprintIdDetails createUnknownFP(final File samFile, final SAMRecord
                         retval.putAll(fingerprintVcf(f));
                     }
 
+                    synchronized (log) {
+                        log.debug("Processed file: " + f.getAbsolutePath() + " (" + filesRead.get() + ")");
+                    }
+
                     if (filesRead.incrementAndGet() % 100 == 0) {
-                        log.info("Processed " + filesRead.get() + " out of " + files.size());
+                        synchronized (log) {
+                            log.info("Processed " + filesRead.get() + " out of " + files.size());
+                        }
                     }
-                } catch(final Exception e){
+                } catch(final Exception e) {
```

Comment:
Should re-format code, missing spaces before `(` here and on line 829 below, and on line 830 after commas.

Reply:
done.